src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js

```diff
@@ -26,8 +26,8 @@ export const BasicSliderInput = (props) => {
     valueLabelDisplay,
     step,
     marks,
-    min,
-    max,
+    min: minValue,
```

Comment:
Almost same as above

Reply:
Here, I'd prefer to keep _minValue_ and _maxValue_ aliases because it's the most clear name and it matches our naming in other files :slightly_smiling_face: